hathor/transaction/storage/transaction_storage.py

```diff
@@ -978,6 +981,14 @@ def is_running_manager(self) -> bool:
         """
         return self.get_value(self._manager_running_attribute) == '1'
 
+    def full_node_crashed(self) -> None:
```

Comment:
Maybe rename to `on_full_node_crash()`.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/809/commits/2938c0855a3efcceece808b8116061e25489dce9

Reply:
Thanks!